api/Makefile

```diff
@@ -39,8 +39,8 @@ vendor:
 .PHONY: test
 test: tidy
 	@echo "Running tests..."
-	go test -v -race -short -cover -coverprofile cover.out ${SRC_ROOT}/... -tags integration
-	go tool cover -func cover.out
+	GOARCH=amd64 go test -race -short -cover -coverprofile cover.out ${SRC_ROOT}/... -tags integration
```

Comment:
Should we make the `GOARCH` value configurable so it's easy to build for other environments too ? Like: https://github.com/caraml-dev/turing/blob/main/engines/experiment/examples/plugins/hardcoded/Dockerfile#L8